src/compute/src/metrics.rs

```diff
@@ -285,6 +285,7 @@ impl WorkerMetrics {
         compatible: bool,
         uncompacted: bool,
         subscribe_free: bool,
+        copy_to_free: bool,
```

Comment:
Consider adding documentation for the parameters, just reading this function it's not clear what they indicate.